app/js/reducers/labOrders/labConceptsReducer.js

```diff
@@ -5,15 +5,36 @@ import {
 } from '../../actions/actionTypes';
 import initialState from '../initialState';
 
+
+const removeDuplicateTests = (tests) => {
+  const uniqueTestIds = Array.from(new Set(tests.map(test => test.uuid)));
+  const uniqueTests = [];
+  for (let i = 0; i < tests.length; i += 1) {
+    const testIndex = uniqueTestIds.indexOf(tests[i].uuid);
```

Comment:
using the `for` loop is not encouraged in es6, please use an iterative array method, preferrably `map`

Reply:
Thanks @Efosaok 
I have updated the function to use `Array.forEach`, `Array.map` won't be able to perform the operation.

Reply:
`map` is more efficient, and why would it not be able to perform the operation